pkg/virt-controller/watch/vmi.go

```diff
@@ -474,6 +477,12 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 		return nil
 	}
 
+	// Clear PodDisruptionBudget label on all completed pods
+	pbdErr := c.clearPBDLabelOnCompletedPods(vmi)
```

Comment:
So uh, you nearly consistently use `pbd` for `PodDisruptionBudget`... shouldn't that be `pdb`?

Reply:
ha, yes. thanks